packages/lwc-services/package.json

```diff
@@ -40,7 +40,7 @@
         "html-webpack-plugin": "~5.3.1",
         "jest": "~26",
         "lwc": "~2.1.1",
-        "lwc-webpack-plugin": "~2.0.0",
+        "lwc-webpack-plugin": "~2.0.2",
```

Comment:
@gamedevsam I'm unsure why you want this. The `~` prefix is for allowing patch updates like this one.

Reply:
I think this is because it's a nested dependency it doesn't resolve patch updates?

Might just need to murder my package-lock file, see if that fixes it. I'll get back to you.

Reply:
Updating the package and deleting `yarn.lock` / `package-lock.json` did the trick, thanks!